exporters/zipkin/src/main/java/io/opentelemetry/exporter/zipkin/ZipkinSpanExporterBuilder.java

```diff
@@ -55,6 +58,19 @@ public ZipkinSpanExporterBuilder setEncoder(BytesEncoder<Span> encoder) {
     return this;
   }
 
+  /**
+   * Sets the Supplier of InetAddress. This Supplier will be used by the {@link
+   * OtelToZipkinSpanTransformer} when creating the Zipkin local endpoint.
```

Comment:
I think we should include a bit of information about what the default value is for this. I have found documentation like that to be super useful, so I don't have to dig into the source code to figure out what the default is.

Reply:
If we document it does that imply that we can't change it? I want us to be able to change it to handle IP address changes eventually.

Reply:
I don't think documenting the current default should preclude changing the default later. Whether we can change the default is an orthogonal decision, IMO.

Reply:
Ok cool then, I'll add a note.